kotlin/internal/opts.bzl

```diff
@@ -49,6 +49,27 @@ _KOPTS = {
         ),
         type = attr.bool,
     ),
+     "x_inline_classes": struct(
+        args = dict(
+            default = False,
+            doc = "Enable experimental inline classes",
+        ),
+        type = attr.bool,
+    ),
+    "x_allow_result_return_type": struct(
+        args = dict(
+            default = False,
+            doc = "Enable result in return types",
+        ),
+        type = attr.bool,
+    ),
+    "x_opt_in": struct(
```

Comment:
See https://github.com/bazelbuild/rules_kotlin/pull/398#pullrequestreview-545091307 -- opt ins appear to be a an area of growth, so the api ought to considered.

Reply:
Let me reopen this PR on a branch - just realized this is off my master